ui/DesignSystem/Component/Wallet/TxList.svelte

```diff
@@ -0,0 +1,18 @@
+<script>
+  import { push } from "svelte-spa-router";
+  import { transactions as store } from "../../../src/transaction.ts";
+  import * as path from "../../../src/path.ts";
+
+  import Remote from "../Remote.svelte";
+  import TransactionListItem from "./TxListItem.svelte";
+
+  const select = event => push(path.transactions(event));
+</script>
+
+<Remote {store} let:data={transactions}>
```

Comment:
It would be good to extract `Remote` here and move it into the screen calling the component, as we are trying to remove any API calls from design system elements.

Reply:
makes sense, need to do that to sort out the org tx's anyway.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/662/commits/2803e98be3a00c0f0594768d94d6039369ba29cb